config/gregtech/worldgen/advanced-rocketry/mars/tungstate_vein.json

```diff
@@ -0,0 +1,37 @@
+{
+  "weight": 10,
+  "density": 0.4,
+  "max_height": 50,
+  "min_height": 20,
+  "surface_stone_material": "lithium",
+  "dimension_filter": ["dimension_id:103"],
+
+  "generator": {
+    "type": "ellipsoid",
+    "radius": [8, 8]
+  },
+  "filler": {
+    "type": "simple",
+    "value": {
+      "type": "weight_random",
+      "values": [
+        {
+          "weight": 20,
+          "value": "ore:scheelite"
+        },
+        {
+          "weight": 60,
+          "value": "ore:tungstate"
+        },
+        {
+          "weight": 20,
+          "value": "ore:lithium"
+        },
+        {
+          "weight": 5,
+          "value": "ore:tellurium"

```

Comment:
Hm, maybe don't directly put tellurium ore on mars. Replace this with something that can be turned into tellurium instead.

Reply:
We don't actually have any ore with a byproduct of Tellurium. The Chalcopyrite process (starts with the thermal centrifuged version) is a source of it as well as Selenium though. Perhaps tellurium ore can be gated until later? The first and only time you need tellurium is for Magneto Resonatic Dust. In this case, it is only first required for UHV components (High Durability Compound Steel). We can put this ore quite far back if we wanted to.